.github/workflows/ci.yml

```
@@ -0,0 +1,36 @@
+name: CI
+
+on: pull_request
```

```suggestion
on: [pull_request, workflow_call]
```

According to @arvin-akbarzadeh this is needed to call that file from `cd.yml`